gatsby-node.js

```diff
@@ -183,3 +186,19 @@ exports.onCreatePage = ({ page, boundActionCreators }) => {
     resolve()
   })
 }
+
+exports.onCreateNode = async ({ node, boundActionCreators }) => {
+  const { createNodeField } = boundActionCreators
+
+  return new Promise((resolve, reject) => {
+    if (node.internal.type === 'InfoYaml') {
```

Comment:
nice - can we do a `switch` on this that would also process the `bio` field of designers?

Reply:
Yes will do